recipe/yum_requirements.txt

```diff
@@ -1,18 +1,37 @@
+libX11
+libXext
+libXrender
+mesa-libGL
+mesa-libEGL
+mesa-dri-drivers
+libXau
 alsa-lib
```

Comment:
What's the benefit of breaking the previously alphabetical order here? A lot of deps just get shuffled around with no actual change?

Reply:
Aside from this nit (and not being the right person to review this): LGTM!